_chapters/02_category.md

```diff
@@ -557,7 +557,7 @@ Incidentally this is the definition of a mathematical law called *commutativity*
 Associativity
 ---
 
-Commutativity law is abided only in contexts where the order is irrelevant i.e. in which an object can be represented as the sum of its parts *when combined in whichever way*. But there are many cases in which an object is to be represented by the sum of its parts, but only when *combined in one specific way*.
+Commutativity law is applicable only in contexts where the order is irrelevant i.e. when an object can be represented as the sum of its parts and therefore *can be combined in whichever way*. But there are many cases in which an object should be represented by the sum of its parts, but only when *combined in one specific way*.
```

Comment:
Here the meaning is changed - in the second case an object also can be represented by the sum of it's parts, the "therefore" is not correct.